### PR TITLE
TAP GUI Maven Artifact Issue for 1.2.0

### PR DESCRIPTION
**Pull request by andreizimin**
Which other branches should this be merged with (if any)?
only for 1.2.0

This is a known issue for 1.2.0 for TAP GUI that we need to mention and add troubleshooting explanation.

### CODE REVIEW

**Comment by RichardJJG** [comment]
Hi, @andreizimin,

1. This seems in direct conflict with earlier PR #1394. Is this PR valid?
2. Is this PR really for the `main` branch or is it, in fact, for `1-2` only?

File: tap-gui/troubleshooting.md [hunk 1]
```diff
@@ -238,4 +237,30 @@ app_config:
...
+
+Source Scan stage vulnerabilities table displays "No records to display" even though there are CVEs in the source scan.
+
+![Screenshot of source scan stage not showing any CVEs](images/troubleshooting-no-cves-source-scan.png)
+
+### Solution
+
+If no CVEs are displayed, run `kubectl describe sourcescan <workload name> -n <workload namespace>` to see if any CVEs were found during the source scan. If so, use the tanzu insight CLI plug-in to query for vulnerability details.  See [Query vulnerabilities, images, and packages](../cli-plugins/insight/query-data.md).
+
+
```
Comment (mstergianis) [inline tap-gui/troubleshooting.md:254]: I see what you mean @RichardJJG. It looks like @andreizimin's branch may have been based on a version that still had the source scan instructions. This section could be deleted

File: tap-gui/troubleshooting.md [hunk 2]
```diff
@@ -238,4 +237,30 @@ app_config:
...
+### Symptom
+
+Source Scan stage vulnerabilities table displays "No records to display" even though there are CVEs in the source scan.
+
+![Screenshot warning of no Maven artifact access ](../images/../tap-gui/images/tap-gui-maven-artifact-1.png)
+
+### Solution
+
+If no CVEs are displayed, run `kubectl describe sourcescan <workload name> -n <workload namespace>` to see if any CVEs were found during the source scan. If so, use the tanzu insight CLI plug-in to query for vulnerability details.  See [Query vulnerabilities, images, and packages](../cli-plugins/insight/query-data.md).
```
Comment (mstergianis) [inline tap-gui/troubleshooting.md:265]: The screenshot has been swapped out but not the instructions

**Review by mstergianis — CHANGES_REQUESTED** [review]

File: tap-gui/troubleshooting.md (hunk 2 above)
Comment (andreizimin) [reply tap-gui/troubleshooting.md:265]: @mstergianis , thank you for making these changes! Are we good to merge?

Comment (mstergianis) [reply tap-gui/troubleshooting.md:265]: Uhh, I haven't made any changes. But I can

Comment (RichardJJG) [reply tap-gui/troubleshooting.md:265]: @mstergianis, are you in the midst of making the requested changes (do you have a local feature branch or similar?) I don't want to merge this prematurely.

Comment (mstergianis) [reply tap-gui/troubleshooting.md:265]: Hey sorry @RichardJJG this one fell through the cracks as I got distracted by other things. I'll take a look at throwing something together today. But it'll be a separate PR. I'm making an assumption here, but I don't think I have push access to Andrei's fork.

Comment (RichardJJG) [reply tap-gui/troubleshooting.md:265]: Perfectly fine to make a separate PR, thanks, @mstergianis

**Comment by RichardJJG** [comment]
Closing because this was covered in a separate PR:
https://github.com/pivotal/docs-tap/pull/1430/files

**Pull request closed without merging**